ext/TOML/src/parser.jl

```diff
@@ -585,7 +585,7 @@ function escape(p::Parser, st::Int, multiline::Bool)
                 NONE()
             end
         else
-            error(p, st, position(p), "unknown string escape: `$("\\x"*hex(ch,2))`")
+            error(p, st, position(p), "unknown string escape: `$("\\x"*string(UInt32(ch), base=16, pad=2))`")
```

Comment:
Maybe assign to a variable on the previous line? This has gotten a bit verbose for splicing inline.